logstash-core/src/test/resources/org/logstash/config/ir/cache/pipeline_reuse_baseline.conf

```diff
@@ -25,7 +26,8 @@ filter {
 }
 output {
   if [a] { noop {} }
+  else { noop {} }
```

Comment:
We the `else` branch because a datasource that wraps an output delegator and evaluates the complement of the condition is already used in the `pipeline_reuse_test.conf` output section.